quinn-proto/src/endpoint.rs

```diff
@@ -575,7 +582,8 @@ impl Endpoint {
         tls: Box<dyn crypto::Session>,
         server_config: Option<Arc<ServerConfig>>,
         transport_config: Arc<TransportConfig>,
-    ) -> (ConnectionHandle, Connection) {
+        ch: ConnectionHandle,
```

Comment:
This feels out of order with respect to our usual style?

Reply:
Fixed.